### PR TITLE
Add Zil support

### PR DESCRIPTION
**Pull request by aranhaagency**
done https://github.com/ensdomains/address-encoder/issues/54

ref: https://github.com/trustwallet/wallet-core/blob/master/tests/Zilliqa/AddressTests.cpp

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -42,6 +42,7 @@ This library currently supports the following cryptocurrencies and address forma
  - BNB (bech32)
  - XLM (ed25519 public key)
  - ATOM (bech32)
+ - ZIL (bech32)
```
Comment (makoto) [inline README.md:45]: Can you provide reference link so that we can verify?

Comment (aranhaagency) [reply README.md:45]: @makoto  https://github.com/trustwallet/wallet-core/blob/master/tests/Zilliqa/AddressTests.cpp

Comment (makoto) [reply README.md:45]: Probably this is better reference https://github.com/Zilliqa/Zilliqa/wiki/Address-Standard#recommended-implementation @Arachnid Do you want them to be in the README of this repo or would it be better to be included in the EIP?

Comment (Arachnid) [reply README.md:45]: @makoto Both would be ideal.

**Review by Arachnid — APPROVED** [review]

**Pull request merged**